packages/zipkin-instrumentation-request/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "zipkin-instrumentation-request",
+  "version": "0.6.2",
```

Comment:
I don't know lerna that well.. when we go to publish, will it know this version hasn't been published yet? or should we add as the last version